DataStructureMemories/DataStructureMemories/ProtocolDataStructure/DataStructMemory.swift

```diff
@@ -0,0 +1,17 @@
+//
+//  DataStructMemory.swift
+//  DataStructureMemories
+//
+//  Created by Alina Zaitseva on 4/17/18.
+//  Copyright © 2018 Alina Zaitseva. All rights reserved.
+//
+
+import Foundation
+
+protocol DataStructMemory {
+    
+    func returnTitleOfString() -> String
```

Comment:
please remove "return" word from the function name

Reply:
got it